tests/test_adapter_plugin.py

```diff
@@ -221,6 +221,54 @@ def test_find_manifest_by_environment_variable(self):
         else:
             del os.environ['OTIO_PLUGIN_MANIFEST_PATH']
 
+    def test_plugin_manifest_order(self):
+        suffix = ".plugin_manifest.json"
+
+        # back up existing manifest
+        bak = otio.plugins.manifest._MANIFEST
+        bak_env = os.environ.get('OTIO_PLUGIN_MANIFEST_PATH')
+
+        local_manifest = {
+            "OTIO_SCHEMA": "PluginManifest.1",
+            "adapters": [
+                {
+                    "OTIO_SCHEMA": "Adapter.1",
+                    "name": "local_json",
+                    "execution_scope": "in process",
+                    "filepath": "example.py",
+                    "suffixes": ["example"]
+                }
+            ],
+        }
+
+        with tempfile.NamedTemporaryFile(suffix=suffix) as otio_path:
+            otio.adapters.write_to_file(local_manifest, otio_path.name, 'otio_json')
+
+            result = otio.plugins.manifest.load_manifest()
+            self.assertTrue(len(result.adapters) > 0)
+            self.assertIn("otio_json", (ml.name for ml in result.adapters))
+            self.assertNotIn("local_otio", (ml.name for ml in result.adapters))
+
+            # set where to find the new manifest
+            os.environ['OTIO_PLUGIN_MANIFEST_PATH'] = otio_path.name
+            result = otio.plugins.manifest.load_manifest()
```

Comment:
This unit test is great.  But a note to self is that it might be nice to have a way to at run time add manifests on the path rather than doing it by manipulating the environment variable and calling `load_manifest()`.